pkg/skaffold/build/custom/custom.go

```diff
@@ -68,9 +69,19 @@ func (b *ArtifactBuilder) Build(ctx context.Context, out io.Writer, a *latest.Ar
 
 func (b *ArtifactBuilder) retrieveCmd(out io.Writer, a *latest.Artifact, tag string) (*exec.Cmd, error) {
 	artifact := a.CustomArtifact
+
+	var cmd *exec.Cmd
 	split := strings.Split(artifact.BuildCommand, " ")
+	if len(split) > 1 {
+		if runtime.GOOS == "windows" {
```

Comment:
also, it feels a little weird to have this check live right in the custom builder implementation. can this maybe be a function `GetShellCommand()` or something in `pkg/skaffold/util/cmd.go`?

Reply:
I didn't want to make things too complicated. I'll what I can do.